scripts/Muon/GUI/Common/plot_widget/plot_widget_presenter.py

```diff
@@ -48,6 +48,8 @@ def __init__(self, view: PlotWidgetViewInterface, model: PlotWidgetModel, contex
         # gui observers
         self._setup_gui_observers()
         self._setup_view_connections()
+        self.enable_observer = PlotWidgetPresenterCommon.EnableObserver(self)
+        self.disable_observer = PlotWidgetPresenterCommon.DisableObserver(self)
```

Comment:
would it not be better to have:
```
self.disable_observer = GenericObservable()
```

Reply:
That was my initial thought and I did try that but it was crashing on starting the Muon GUI. I thought it may be to do with it not being a tab so I followed the same structure as the load widget (also not a tab)

Reply:
it shouldnt care if it is a tab or not. The load widget used that method because the generic ones did not exist yet

Reply:
Ah then that may have been the same problem, it was complaining None type does not have x method

Reply:
it might just be that things are created in the wrong order

Reply:
I am not sure what the problem was before maybe something in how I had it set up, but I have just tried generic observers and it seems to work fine